automation_tools/__init__.py

```diff
@@ -964,6 +964,19 @@ def product_install(distribution, create_vm=False, certificate_url=None,
             host=host
         )
 
+    execute(fix_qdrouterd_listen_to_ipv6, host=host)
+
+
+def fix_qdrouterd_listen_to_ipv6():
+    """Configure qdrouterd to listen to IPv6 instead of IPv4.
+
+    Workaround for BZ #1219902.
+
+    """
+    run('sed -i -e "s|addr: 0.0.0.0|addr: ::|" '
```

Comment:
I tried replacing both and it was working. But, of course, I'm not sure of any new issues either. I'll better replace only the first occurrence, thanks for suggestion.